src/ep.config.json

```diff
@@ -54,7 +54,7 @@
   "gaTrackingId": "",
   "arKit": {
     "enable": true,
-    "skuArImagesUrl": "https://s3.amazonaws.com/referenceexp/ar/%sku%.usdz"
+    "skuArImagesUrl": "https://s3.amazonaws.com/referenceexp/ar/%sku%.usdz#callToAction=Add%20to%20cart&checkoutTitle=%title%&checkoutSubtitle=%description%&price=%price%"
```

Comment:
Can we append this to the usdz in our code instead of at the config level? Will keep things cleaner and maintain configs much easier.

Reply:
If we move it to our code, we'll need to externalize the "Add to cart" string into the locale files as well